src/config.xml.mustache

```diff
@@ -193,6 +193,10 @@
                 <application android:exported="true" />
             </edit-config>
 
+            <edit-config file="app/src/main/AndroidManifest.xml" mode="merge" target="/manifest/application">
+                <application android:usesCleartextTraffic="true" android:requestLegacyExternalStorage="true"/>
```

Comment:
Is the `usesClearTraffic` needed? Iiuc this is not needed when when are communicating over https (which is a requirement for the cookies to persist).